### DISCUSSION TITLE
Add Hot Spot sensor for RTX, actually it is not detected

### DISCUSSION THREAD

**Discussion by juanjoworld** [Rem0o/FanControl.Releases#2695] [Ideas]
Actually the sensor is not detected by fan control, but it is a critical sensor becaues take into account temeperatures in other places of the GPU Board what not is the core. GPU-z detect the sensor if you what to see it, In order to correctly control the GPU Fans this sensor is critical.

**Comment by Rem0o** [comment]
In Sensor sources, untick NVApiWrapper, tick GPU.   Hotspot will be in there.

**Reply by juanjoworld** [reply]
Thank you very much, i am going now to do some curves and mix hehe to take all into account
